be/src/runtime/stream_load/stream_load_context.h

```diff
@@ -113,7 +113,9 @@ class StreamLoadContext {
     bool unref() { return _refs.fetch_sub(1) == 1; }
 
 public:
-    MemTracker* mem_tracker = nullptr;
+    std::shared_ptr<RuntimeProfile> runtime_profile;
+    std::shared_ptr<MemTracker> query_mem_tracker;
+    std::shared_ptr<MemTracker> instance_mem_tracker;
```

Comment:
Should add comment for why using shared_ptr.

Reply:
Fixed